src/components/repeat/ohri-repeat.component.tsx

```diff
@@ -115,46 +106,57 @@ export const OHRIRepeat: React.FC<OHRIFormFieldProps> = ({ question, onChange })
   };
   const nodes = questions.map((question, index) => {
     const deleteControl =
-      index !== 0 ? (
-        <Column style={{ paddingTop: '1.2rem', marginLeft: '.5rem' }}>
-          <Button
-            renderIcon={() => <TrashCan size={32} />}
-            kind="danger--tertiary"
-            onClick={() => removeQuestion(question)}
-            hasIconOnly
-          />
-        </Column>
+      questions.length > 1 ? (
+        <div>
```

Comment:
Added wrapper div to fix:

<img width="345" alt="Screenshot 2022-11-08 at 12 15 29" src="https://user-images.githubusercontent.com/26084581/200540994-74b7a000-65ae-4b23-8d6b-655795c11d11.png">